api/proto/config/devmodel.proto

```diff
@@ -68,11 +68,8 @@ message SystemAdapter {
         string networkUUID = 4;
```

Comment:
Would make sense to add a comment that if this is not set, then the device will treat the port as an app-shared L2 port i.e., the device will not configure any IP on the port.